prow/config/presubmits.yaml

```diff
@@ -908,7 +742,7 @@ presubmits:
     skip_report: false
     max_concurrency: 2
     branches:
-      - ^maistra-2.1$
```

Comment:
This and the other similar changes should stay. We should always use `^$` to avoid branches with similar names triggering the wrong jobs.

Reply:
I agree, will exclude that from this PR.